src/lib/Libifl/pbs_loadconf.c

```diff
@@ -314,7 +319,10 @@ parse_psi(char *conf_value)
 	}
 	free_string_array(list);
 	pbs_conf.pbs_num_servers = i;
-	pbs_conf.psi_str = strdup(conf_value);
+	pbs_conf.psi_str = strdup(local_conf);
```

Comment:
Suggestion: Instead of calling strdup() all the time, only call it when conf_value != NULL.  Otherwise just assign psi_str = local_conf.  It's already allocated memory.  Then you can remove the free below.

Reply:
Thanks for the suggestion, but i feel like the code will be simpler if we just do the strdup and free local_conf (actually keep it local). So I'd prefer that we leave it as is unless you feel strongly about this.

Reply:
I guess it is a bit simpler.  I just didn't like the idea of duping allocated memory, just to free the memory.  It doesn't really matter because local_conf should be just a few bytes, and this is called once at the start of the program.